bemuse/src/app/index.js

```diff
@@ -124,14 +122,9 @@ function shouldActivateServiceWorker() {
 function setupServiceWorker() {
   if (!('serviceWorker' in navigator)) return false
   if (!shouldActivateServiceWorker()) return false
-  registerServiceWorker()
```

Comment:
Without a service worker it breaks Bemuse’s offline capabilities…

- todo: Bring back service worker so that Bemuse can continue to work offline.
- todo: Add a Playwright test to ensure Bemuse works offline. [Example](https://github.com/dtinth/ppqr.app/blob/master/tests/ppqr.spec.ts#L44)

LMK if you’d like to help with this, otherwise I’ll resolve that when I come around to it.

Reply:
Sorry for lack of migrating Service Worker. I will fix as soon as possible.

Reply:
Fixed on 7a02951 and Added the test on 441d20a.